content/Developer_Program/WebAuthn_Starter_Kit/Attestation.adoc

```diff
@@ -0,0 +1,90 @@
+
+In some instances, if an attestation certificate is self-signed, then there may be a generally diminished level of assurance than would be the case with a certificate signed by a trusted manufacturer such as https://www.yubico.com/[Yubico] or a known root CA. As a result, an RP may choose to pre-configure a warning to users during registration that a nominated device is not trusted according to its sources. 
+
+Furthermore, for some heavily regulated services such as those in the financial industry or the public sector, an RP could be required to know more about the devices that are accessing it so that it may enforce stricter policies. Due to the extremely sensitive nature of the underlying data, these services must guarantee that cryptographic keys are secure, that biometrics are of a certain level of accuracy or that devices have been adequately certified, which can only be achieved if the registration request is coming from a known and trusted model of FIDO authenticator.
+
+=== What are the use cases for attestation?
+
+During a WebAuthn account registration ceremony, the unique model number of a nominated security device is sent to the RP alongside the newly created public key used to access that specific service (not to be mistaken for the attestation key). The unique model number is also known as the “Authenticator Attestation Globally Unique ID” (AAGUID), and upon receiving this unique identifier, the RP is able to look up a metadata statement from a service such as the https://fidoalliance.org/metadata/[FIDO Metadata Service (MDS)] or potentially a https://developers.yubico.com/U2F/yubico-metadata.json[JSON configuration file] to complete the discovery of the nominated security device. Every record within these services will have an AAGUID that corresponds to a known device.
+
+Assuming that the signature associated with the attestation certificate containing the public key is correct and that the certificate chain is validated to a trusted CA, a service can then use the collected metadata about the device for one of several use cases.
+
+image::Images/attest-figure3.png[align="center"]
+[.text-center]
+*Figure 3 - Attestation provides the option to allow or deny any security devices based on the AAGUID or Attestation Certificate*
+[.text-left]
+
+The first of which is an "`allow list"`, whereby only a limited and pre-defined subset of all security devices may be registered and used as an authenticator in conjunction with a particular service. Depending on policies and how broad the array of security devices is amongst the userbase, this may be the most secure option to weed out potential issues created by untrusted security devices. Allowed FIDO authenticators are usually those that have been tested internally or during development and have been vetted as a secure and compatible option.
```

Comment:
@sebastianelfors Would it make sense to add and example of a policy, say the FPIS requirement, as the driver for the need for an allow list here?